lib/util/git/index.ts

```diff
@@ -636,21 +639,48 @@ export async function isBranchModified(branchName: string): Promise<boolean> {
     return isModified;
   }
 
-  logger.debug('branch.isModified(): using git to calculate');
-
   await syncGit();
-  // Retrieve the author of the most recent commit
-  let lastAuthor: string | undefined;
+  // Retrieve the commit authors
+  let branchAuthors: string[] = [];
   try {
-    lastAuthor = (
-      await git.raw([
-        'log',
-        '-1',
-        '--pretty=format:%ae',
-        `origin/${branchName}`,
-        '--',
-      ])
-    ).trim();
+    if (baseBranch) {
+      logger.debug(
+        `branch.isModified(): using git to calculate authors between ${branchName} and ${baseBranch}`
+      );
+      branchAuthors = [
+        ...new Set(
+          (
+            await git.raw([
+              'log',
+              '--pretty=format:%ae',
+              `origin/${branchName}..origin/${baseBranch}`,
+              '--',
+            ])
+          )
+            .trim()
+            .split('\n')
+        ),
+      ];
```

Comment:
```suggestion
      ];
      if (branchAuthors.length > 1) {
        logger.debug(`Found multiple git authors in branch: ${branchAuthors.join(", ")}`);
      }
```

I think this should not be the common case, so worth mentioning?